File: JAVA/removeElement.java
package JAVA;

import java.util.Scanner;

public class removeElement {
    public static void main(String[] args) {
        int n, i, ele;
        Scanner t = new Scanner(System.in);
        System.out.println("Enter length of array and element ot be removed");
        n = t.nextInt();
        ele = t.nextInt();
        int arr[]  = new int[n];
        System.out.println("Enter element of array");
        for(i = 0; i < n; i++) {
            arr[i] = t.nextInt();
        }
        Duplicate d = new Duplicate();
        int len = d.remove(arr,ele);
        System.out.println("Length of Array is " + len);
    }
}
class Duplicate {
    public int remove(int [] A, int ele) {
        if(A==null)
        return 0;
        else {
            int i = 0;
            int j = 0;
            while(i < A.length) {
                if(A[i] != ele) {
                    j++;
                }
                i++;
            }
            return j;
        }
    }
}

File: C/removeElement.c
// REMOVE DUPLICATE ELEMENT IN  ARRAY
#include<stdio.h>
#include<stdlib.h>

int removeElement(int arr[], int n, int ele) {
    if(n==1) 
    return 1;
    else {
        int i, j;
        j = 0;
        i = 0;
        while(j < n) {
            if(arr[j] != ele) {
                i++;
            } 
            j++;
        }
        return i;
    }
    
}


int main() {

    int n, ele, *A;
    printf("Enter number of array and element to remove\n");
    scanf("%d%d", &n, &ele);
    A = (int *)malloc(sizeof(int)*n);
    printf("Enter element in Array\n");
    for(int i = 0; i < n; i++) {
        scanf("%d", &A[i]);
    }
    
    
    int length = removeElement(A,n,ele);
    printf("Length of array is: %d \n", length);


    return 0;
}